src/install.ts

```diff
@@ -25,8 +25,8 @@ export const getInstallIdByTags = async (tags: IEveryAuthTagSet): Promise<string
   const installs = await getChildrenByTags<IEveryAuthInstall>(
     {
       [SERVICE_TAG]: tags.serviceId,
-      [USER_TAG]: tags.userId,
-      [TENANT_TAG]: tags.tenantId || tags.userId,
+      ...((tags.userId !== undefined && { [USER_TAG]: tags.userId, [TENANT_TAG]: tags.userId }) || {}),
+      ...((tags.tenantId !== undefined && { [TENANT_TAG]: tags.tenantId }) || {}),
```

Comment:
Let's make sure these behaviors are covered by the test cases, so that the expected behaviors are obvious and enforced via test code.

Reply:
Added tests. Also changed `pageSize` to `count` otherwise paging was hosed.